test/e2e/util.go

```diff
@@ -161,6 +161,10 @@ type TestContextType struct {
 	GatherMetricsAfterTest            bool
 	// Currently supported values are 'hr' for human-readable and 'json'. It's a comma separated list.
 	OutputPrintType string
+	// CreateTestingNS is responsible for creating namespace used for executing e2e tests.
+	// It accepts namespace base name, which will be prepended with e2e prefix, kube client
+	// and labels to be applied to a namespace.
+	CreateTestingNS func(baseName string, c *client.Client, labels map[string]string) (*api.Namespace, error)
```

Comment:
Pretty sure tests should not be calling CreateTestingNS directly, but should be getting it from the framework. I think @smarterclayton has squashed those several times and they keep popping back up